src/test/kotlin/com/atlassian/performance/tools/virtualusers/lib/infrastructure/Jperf425WorkaroundMysqlDatabase.kt

```diff
@@ -25,7 +25,7 @@ class Jperf425WorkaroundMysqlDatabase(
 }
 
 /**
- * Copy-pasted verbatim from [infrastructure:4.11.0](https://github.com/atlassian/infrastructure/blob/release-4.11.0/src/main/kotlin/com/atlassian/performance/tools/infrastructure/Docker.kt)
+ * Copy-pasted verbatim from [infrastructure](https://github.com/atlassian/infrastructure/blob/master/src/main/kotlin/com/atlassian/performance/tools/infrastructure/Docker.kt)
```

Comment:
`master` is a moving target, but this copy-paste is static.

Reply:
Well, the usefulness of this comment is limited to the point where you have to update the Docker copy, I don't think including the version number makes it more useful.

Reply:
Until it starts lying. Whenever `master` moves forward, this comment is no longer accurate. The bigger the gap from the actual source to `master`, the bigger the misleading. Especially if `master` goes to 5.x.y and e.g. nukes the `internal class Docker` out of existence.